### PR TITLE
Es6 modules

### PR DESCRIPTION
**Pull request by AbbyNyakara**
Refactor the initial code for the awesome books project. Use Luxon library for the dates.

### CODE REVIEW

File: modules/Showbooks.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    `;
+
+      // Reset the values to null to allow the user to input again
+      title.value = '';
+      author.value = '';
+    }
+  }
+
+  // Remove book
+  const removeBook = (index) => {
```
Comment (Ismailco) [inline modules/Showbooks.js]: - [x] The `removeBook` function is not working properly, please fix this issue so your app works as expected. in the browser console it returns the error: 
    ```
    Uncaught ReferenceError: removeBook is not defined
        at HTMLButtonElement.onclick
    ```

File: modules/Showbooks.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+const title = document.querySelector('.title-input');
+const author = document.querySelector('.author-input');
+const loggedBooks = document.querySelector('.logged-books');
+const addButton = document.querySelector('.add-book');
+const contact = document.querySelector('.contact');
+const form = document.querySelector('form');
+
+let books = [];
+
+const activateBooks = () => {
```
Comment (Ismailco) [inline modules/Showbooks.js]: - [x] [OPTIONAL] You can divide your code more into separate files and import them as modules, the more the better for the sake of practice

**Review by Ismailco — CHANGES_REQUESTED** [review]
### Hi @AbbyNyakara,

Good job so far 👍🏼 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️ No linter errors
✔️ Used Github Flow
✔️ Work documented professionally
✔️ Clean code

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: modules/Showbooks.js (hunk 1 above)
Comment (AbbyNyakara) [reply modules/Showbooks.js]: I refactored the code to use classes instead. Introducing the module type introduces a bug

**Review by andersonlebon — APPROVED** [review]
## Hi @AbbyNyakara 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 :clap: :clap: :100: 

nodding-yes

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Your README file is professional :heavy_check_mark: 

## Optional suggestions
- [ ] Kindly Check the comments under the review._
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**